lib/src/imgstorage.rs

```diff
@@ -139,14 +139,15 @@ impl Storage {
     #[context("Creating imgstorage")]
     pub(crate) fn create(sysroot: &Dir, run: &Dir) -> Result<Self> {
         Self::init_globals()?;
-        let subpath = Utf8Path::new(SUBPATH);
+        let subpath = Self::subpath();
+
         // SAFETY: We know there's a parent
         let parent = subpath.parent().unwrap();
         if !sysroot
-            .try_exists(subpath)
+            .try_exists(&subpath)
```

Comment:
Also minor style: I tend to prefer doing `&` on the variable like `let subpath = &Self::subpath()`